conclave-host/src/main/kotlin/com/r3/conclave/host/internal/SocketHostEnclaveInterface.kt

```diff
@@ -22,15 +22,12 @@ import java.util.concurrent.Semaphore
  *  - Handle the low-level details of the messaging protocol (socket with streamed ECalls and OCalls).
  */
 class SocketHostEnclaveInterface(port: Int = 0, private val maxConcurrentCalls: Int) : HostEnclaveInterface(), Closeable {
-    private val serverSocket = ServerSocket(port)
+    private val serverSocket = ServerSocket(port, 100)
```

Comment:
magic number alert! Why are you having to increase the backlog value?

Reply:
Another one that slipped through! Should just be set to the max number of concurrent calls. Will fix.

Reply:
Actually, as the socket creation process is synchronous, it can be left default. Will remove.